shared/login/register/error/index.render.desktop.js

```diff
@@ -8,8 +8,9 @@ import {ConstantsStatusCode} from '../../../constants/types/flow-types'
 import {Text} from '../../../common-adapters'
 
 const renderError = (error: RPCError) => {
-  const fields = (error.fields || []).reduce((acc, f) => {
-    acc[f.key] = f.value
+  const fields = (Array.isArray(error.fields) ? error.fields : []).reduce((acc, f) => {
+    const k = f && typeof f.key === 'string' ? f.key : ''
```

Comment:
:sheep: This is a little wonky (falling back to the key of `''`) but probably not worth complicating it more. Flow isn't really helping here :/